packages/terra-utils/lib/index.ts

```diff
@@ -0,0 +1,87 @@
+import { Block, SwapParams, Transaction, terra } from '@liquality/types'
+import { addressToString, validateExpiration, validateSecretHash, validateValue } from '@liquality/utils'
+import { InvalidAddressError } from '@liquality/errors'
+
+export const normalizeBlock = (data: any): Block => ({
+  hash: data.block_id.hash,
+  timestamp: convertDateToTimestamp(data.block.header.time),
+  size: Number(data.block.header.height),
+  number: Number(data.block.header.height),
+  parentHash: data.block.last_commit.block_id.hash
+})
+
+export const normalizeTransaction = (data: any, asset: string): Transaction<terra.InputTransaction> => {
```

Comment:
I could not the number of confirmations, can you include it please

Reply:
Yes I'm thinking about them as well because they are needed in the wallet but the transaction it self do not have confirmations prop, [example tx](https://finder.terra.money/tequila-0004/tx/7A58E4C26FC3A3E9883942D89380CE0143275088BE111849BDE8638C34C80EC6), maybe we can depend on status prop or i'll need to count block which are passed from broadcasting tx and if they are `gte(10)` to set specific number of confirmations

Reply:
`confirmations = currentBlock - txBlock`